lib/loaders/express.loader.ts

```diff
@@ -48,6 +48,14 @@ export class ExpressLoader extends AbstractLoader {
         app.use(express.static(clientPath, options.serveStaticOptions));
         app.get(options.renderPath, renderFn);
       }
+
+      app.use((err: Error, req, res, next: Function) => {
+        if (err.message.includes('ENOENT')) {
```

Comment:
What if the `message` property is `undefined`?

Reply:
great point. I will fix that.